# a vpPoint

int32 i
int32 j
float64 x
float64 y
float64 z

